src/base/utils/misc.cpp

```diff
@@ -503,9 +503,9 @@ QList<bool> Utils::Misc::boolListfromStringList(const QStringList &l)
 
 bool Utils::Misc::isUrl(const QString &s)
 {
-    const QString scheme = QUrl(s).scheme();
-    QRegExp is_url("http[s]?|ftp", Qt::CaseInsensitive);
-    return is_url.exactMatch(scheme);
+    static const QRegExp isURL("http[s]?|ftp", Qt::CaseInsensitive);
```

Comment:
If you care about performance you might want to use `QRegularExpression` instead of making it `static`.

>The QRegularExpression class introduced in Qt 5 is a big improvement upon QRegExp, in terms of APIs offered, supported pattern syntax and speed of execution.